examples/promise-co-await/co.js

```diff
@@ -1,8 +1,8 @@
-var mysql = require('../promise.js');
+var mysql = require('../../promise.js');
```

Comment:
though one thing is that the examples wont work out of the box -- ie I just cloned the repo and ran node examples/promise-co-await/co.js -- works like a charm. If we use absolute requires you wont be able to do that. Happy to change stuff as long as you think that's alright!